### PR TITLE
Add the release action to the github workflow

### PR DESCRIPTION
**Pull request by Tatskaari**

### CODE REVIEW

**Review by peterebden — APPROVED** [review]

File: resolve/driver/please_driver.go [hunk 1]
```diff
@@ -134,6 +134,10 @@ func (driver *pleaseDriver) loadPackage(info *packageInfo) error {
 		return nil
 	}
 
+	if knownimports.IsInGoRoot(info.id) {
```
Comment (Tatskaari) [inline resolve/driver/please_driver.go:137]: This fixes the issue where we can't resolve golang.org/x/crypto

File: package/BUILD [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+def cross_compile(arch):
+    return build_rule(
+        name = f"go_deps_{arch}",
+        srcs = [f"///{arch}//:go-deps"],
+        outs = [f"go-deps-{VERSION}-{arch}"],
+        cmd = "mv $SRC $OUT",
+    )
+
+filegroup(
+    name = "releaes_files",
```
Comment (peterebden) [inline package/BUILD:12]: releaes -> release

**Review by peterebden — CHANGES_REQUESTED** [review]

File: package/BUILD (hunk 2 above)
Comment (tiagovtristao) [inline package/BUILD:12]: typo

Comment (Tatskaari) [reply package/BUILD:12]: It's a stylistic choice ;P

**Pull request merged**